docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,106 @@
+# Percona Monitoring and Management 2.28.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+    <th class="field-name">Date:</th>
+    <td class="field-body">May 12, 2022</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of the PMM instance. This ensures that you access the latest features of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+### Advisors
+
+
+- **Advisor checks enabled by default**
+
+    Starting with the previous release and continuing with this one, we have added significant improvements to the Advisors Checks functionality in performance, assessment coverage, and user experience.
+    
+    As a mature and generally useful feature, this option is now enabled by default for easier access to automatic checks and better insight into database health and performance, delivered by Percona Platform.
+    
+    !!! note alert alert-primary "Note"
+        Upgrading to PMM will automatically enable this feature for existing PMM instances. You can disable it at any time from your PMM dashboard on the **Advanced Settings** page.
+
+- **Run individual advisor checks**
+
+    In addition to running all available advisors at once, you now have the option to run each advisor check individually. 
+
+    This gives you more granular control over the checks on your connected databases. Running checks individually also means that you get the results for relevant advisors faster and that you can focus on resolving failed checks one at a time. For more information, see [Working with Advisor checks](https://docs.percona.com/percona-monitoring-and-management/how-to/advisors.html).
+
+
+- **Enhanced Advisor checks**
+
+    PMM 2.28 includes a new major version of Advisors that features some important enhancements. The most significant changes are:
+
+    * Support for multiple queries
+    * Support for Victoria Metrics as a data source
+
+    In a nutshell, these are changes will allow experts to create more intelligent advisor checks so that we can continue delivering more value to your connected PMM instances.
```

Comment:
```suggestion
    In a nutshell, these changes will allow experts to create more intelligent advisor checks so that we can continue delivering more value to your connected PMM instances.
```